### PR TITLE
Delete activities functionality implemented

### PR DESCRIPTION
**Pull request by harvli**
merged with dev aka with alex's changes in the previous pull request

### CODE REVIEW

**Review by jbhowat — APPROVED** [review]
Super work!

File: src/containers/ActivityInfo.jsx [hunk 1]
```diff
@@ -10,40 +10,65 @@ const ActivityInfo = (props) => {
...
+        console.log('targetedAct', targetedAct[0].activity);
+        console.log('teamID', team.team_id);
+        // Harvey: Send delete request to server to delete targeted card
+        fetch('/db/deleteActivity', {
+          method: 'DELETE',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            activityName: targetedAct[0].activity, // 'Go to a local thrift shop'
```
Comment (atu816) [inline src/containers/ActivityInfo.jsx:45]: I think test comments can be omitted in the future

Comment (harvli) [reply src/containers/ActivityInfo.jsx:45]: Agreed, noted!

**Review by atu816 — APPROVED** [review]
Nice cleanup and additions! Some pseudo comment like test comments can be omitted and other thoughts I think can be summarized to be more concise. Your code is good and explains half of it on its own :)

**Pull request merged**